### PR TITLE
Removed unused RunSequence from gulpfile.js and fixed a bunch of typos in index.html

### PR DESCRIPTION
**Pull request by tlakomy**
In the pursuit of getting a free meet.js summit ticket (:onion:) I've taken a quick look through the codebase and I've noticed that `runSequence` is imported in `gulpfile.js` and is not used at all. I've also fixed a small typo in `index.html`.

EDIT: I noticed that polish city names were not consistent in `index.html`: Cracow/Kraków, Gdansk/Gdańsk etc. I noticed that polish characters are kept in other sections so I've made it consitent.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -388,7 +388,7 @@ <h2 class="code__heading">#code-of-conduct</h2>
         <h2 class="about__heading">#about-us</h2>
         <p class="about__text">meet.js is the name for JavaScript meetups in Poland, organised for programmers
           who are interested in JavaScript, HTML and CSS. Our events have been organised for over 6 years now in 11
-          major business hubs in the country (Gdansk, Warsaw, Cracow, Poznan, Wroclaw, Lodz, Katowice,
+          major business hubs in the country (Gdańsk, Warsaw, Kraków, Poznań, Wroclaw, Łodz, Katowice,
```
Comment (feimosi) [inline src/index.html]: _Wrocław_
_Łódz_
:slightly_smiling_face:

Comment (tlakomy) [reply src/index.html]: Good point 👍

**Comment by tlakomy** [comment]
On the other hand, Warsaw and Cracow are waaay more familiar for foreigners so perhaps they should both remain in English? 🤔

**Comment by feimosi** [comment]
I agree, Warsaw and Cracow are more 'foreigners-friendly' names.

**Comment by typeofweb** [comment]
Why change Cracow to Kraków but keep Warsaw instead of Warszawa? It's either one way or another and personally I think we should go with Warsaw and Cracow.

**Comment by typeofweb** [comment]
Is it possible to remove `run-sequence` from `package.json` too?

**Review by typeofweb — CHANGES_REQUESTED** [review]
See my comments

**Comment by tlakomy** [comment]
@mmiszy comments addressed 👍

**Review by lukaszklis — APPROVED** [review]

**Pull request merged**